CHANGELOG.md

```diff
@@ -1,1406 +1,2637 @@
-  * #### Changed
-    * Renamed `create_waveform_f64` -> `create_waveform`
-  * #### Removed
-    * `create_waveform_i16`
-    * `write_binary16_waveform`: Use `write_waveform`
-    * `write_named_waveform_i16`: Use `write_waveform`
-    * `write_named_waveform_f64`: Use `write_waveform`
-* ### `niscope` (NI-SCOPE)
-  * #### Added
-    * `fetch_into` for high-performance fetch using numpy.array. Supported element types:
-      * `numpy.float64`
-      * `numpy.int8`
-      * `numpy.int16`
-      * `numpy.int32`
-  * #### Changed
-    * Added default values for timeout on all fetch and read functions.
-    * Property input_impedance no longer uses enum InputImpedance.
-  * #### Removed
-    * `AddWaveformProcessing` - See #667 for rationale
-    * `ClearWaveformProcessing` - See #667 for rationale
-    * `FetchArrayMeasurement` - See #667 for rationale
-
-## 0.5.0 - 2017-11-27
-* ### ALL
-  * #### Removed
-    * enum definitions that are not referenced by a function and/or an attributes
-* ### `nidmm` (NI-DMM)
-  * #### Added
-    * `get_ext_cal_recommended_interval`
-* ### `nidcpower` (NI-DCPower)
-  * #### Added
-    * `get_ext_cal_last_date_and_time`
-    * `get_ext_cal_last_temp`
-    * `get_ext_cal_recommended_interval`
-    * `measure_multiple`
-* ### `nifgen` (NI-FGEN)
-  * #### Removed
-    * `adjust_sample_clock_relative_delay`
-* ### `niscope` (NI-SCOPE)
-  * #### Added
-    * Initial release
-  * #### Removed
-    * Removed Peer to Peer attributes
-
-## 0.4.0 - 2017-11-07
-* ### ALL
-  * #### Changed
-    * Simplified examples by removing try/except
-    * **SOURCE BREAKER:** Changed names of enum value names to correspond to C #defines
-* ### `nidmm` (NI-DMM)
-  * #### Changed
-    * Removed incorrect leading underscore from some enum values:
-      * `Function.AC_VOLTS_DC_COUPLED`
-      * `Function.WAVEFORM_CURRENT`
-      * `MeasurementCompleteDest.LBR_TRIG_0`
-      * `OperationMode.IVIDMM_MODE`
-      * `SampleTrigger.EXTERNAL`
-      * `SampleTrigger.TTL_3`
-      * `TriggerSource.TTL_0`
-      * `TriggerSource.TTL_3`
-      * `TriggerSource.TTL_7`
-      * `TriggerSource.PXI_STAR`
-* ### `niswitch` (NI-SWITCH)
-  * #### Removed
-    * Support for `is_debounced` and `is_scanning` functions. Instead use the attribute of the same name.
-* ### `nidcpower` (NI-DCPower)
-  * #### Added
-    * New example `nidcpower_advanced_sequence.py`
-  * #### Changed
-    * Fixed method signature for:
-      * `wait_for_event`
-      * `create_sequence`
-      * `create_advanced_sequence`
-  * #### Removed
-    * Support for `measure_multiple` until issue #444 is addressed.
-* ### `nifgen` (NI-FGEN)
-  * #### Added
-    * Initial release
-
-## 0.3.0 - 2017-10-13
-* ### ALL
-  * #### Added
-    * Support for ViInt64 (64-bit integers)
-  * #### Changed
-    * Modified how methods with repeated capabilities are invoked. There's no longer (for example) a `channel_name` input. Instead:
-      ```python
+---
+
+### nidcpower (NI-DCPOWER)
```

Comment:
Please take care to spell the name for the products correctly.
NI-DCPower.
NI Switch Executive
etc.

The README has it right.

This is customer facing.